index.html

```diff
@@ -1,37 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <script src="https://kit.fontawesome.com/60f96566f6.js" crossorigin="anonymous"></script>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <script src="https://kit.fontawesome.com/60f96566f6.js" crossorigin="anonymous"></script>
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=G-D8J3XCGK09"></script>
     <script>
-      window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
+
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
 
       gtag('config', 'G-D8J3XCGK09');
     </script>
     <title>Enmanuel's Portfolio</title>
+        <script src="script.js"></script>
 </head>
 <body>
     <header class="header">
-            <img src="elements/my-logo.png" alt="logo" class="img-logo">            
+        <img src="elements/my-logo.png" alt="logo" class="img-logo">            
         <nav class="nav">
             <ul class="navbar">
                 <li><a href="#">portfolio</a></li>
                 <li><a href="#">about</a></li>
                 <li><a href="#">contact</a></li>
             </ul>       
-            <i class="fas fa-bars icon-element"></i>
+            <i onclick="showMenu()" id="menu-button" class="fas fa-bars icon-element"></i>
         </nav>
-     
+        
     </header>
+    <div id="mobile-menu" class="mobile-menu display">
+        <ul>
+            <li class="menu-links close-button"><a onclick="showMenu()" href="#">X</a></li>
+            <li class="menu-links"><a href="#">Portfolio</a></li>
+            <li class="menu-links"><a href="#">About</a></li>
+            <li class="menu-links"><a href="#">Contact</a></li>
+        </ul>
+    </div>
```

Comment:
* When i click (or tap) any option of the mobile menu, the mobile menu should disappear as stated in the project requirement